.env

```
@@ -0,0 +1,44 @@
+# Database Configuration
+MONGO_URI="mongodb+srv://eminchniti_db_user:XyGRnmNUb8ezw4MS@cluster0.ygmz3nl.mongodb.net/Accountia?appName=Cluster0"
+
+# JWT Configuration
+JWT_SECRET=your-super-secret-jwt-key-change-this-in-production
+
+# Email Configuration (Gmail SMTP)
+GMAIL_USERNAME="gordian.auth@gmail.com"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)